command/config/config.go

```diff
@@ -326,16 +326,17 @@ type EnvConfig struct {
 	}
 
 	Settings struct {
-		DefaultDriver        string `envconfig:"DRONE_DEFAULT_DRIVER" default:"amazon"`
-		ReusePool            bool   `envconfig:"DRONE_REUSE_POOL" default:"false"`
-		BusyMaxAge           int64  `envconfig:"DRONE_SETTINGS_BUSY_MAX_AGE" default:"24"`
-		FreeMaxAge           int64  `envconfig:"DRONE_SETTINGS_FREE_MAX_AGE" default:"720"`
-		MinPoolSize          int    `envconfig:"DRONE_MIN_POOL_SIZE" default:"1"`
-		MaxPoolSize          int    `envconfig:"DRONE_MAX_POOL_SIZE" default:"2"`
-		EnableAutoPool       bool   `envconfig:"DRONE_ENABLE_AUTO_POOL" default:"false"`
-		HarnessTestBinaryURI string `envconfig:"DRONE_HARNESS_TEST_BINARY_URI"`
-		PluginBinaryURI      string `envconfig:"DRONE_PLUGIN_BINARY_URI" default:"https://github.com/drone/plugin/releases/download/v0.3.6-beta"`
-		PurgerTime           int64  `envconfig:"DRONE_PURGER_TIME_MINUTES" default:"30"`
+		DefaultDriver          string `envconfig:"DRONE_DEFAULT_DRIVER" default:"amazon"`
+		ReusePool              bool   `envconfig:"DRONE_REUSE_POOL" default:"false"`
+		BusyMaxAge             int64  `envconfig:"DRONE_SETTINGS_BUSY_MAX_AGE" default:"24"`
+		FreeMaxAge             int64  `envconfig:"DRONE_SETTINGS_FREE_MAX_AGE" default:"720"`
+		MinPoolSize            int    `envconfig:"DRONE_MIN_POOL_SIZE" default:"1"`
+		MaxPoolSize            int    `envconfig:"DRONE_MAX_POOL_SIZE" default:"2"`
+		EnableAutoPool         bool   `envconfig:"DRONE_ENABLE_AUTO_POOL" default:"false"`
+		HarnessTestBinaryURI   string `envconfig:"DRONE_HARNESS_TEST_BINARY_URI"`
+		PluginBinaryURI        string `envconfig:"DRONE_PLUGIN_BINARY_URI" default:"https://github.com/drone/plugin/releases/download/v0.3.6-beta"`
+		PurgerTime             int64  `envconfig:"DRONE_PURGER_TIME_MINUTES" default:"30"`
+		AutoInjectionBinaryURI string `envconfig:"DRONE_AUTO_INJECTION_BINARY_URI" default:"https://github.com/sahithibanda01/auto-build-detection/releases/download/v1.0.9"`
```

Comment:
Use the right repo and make sure there exists a pipeline which generates and publishes these binaries properly.